docs/virtual_machines/instancetypes.md

```diff
@@ -0,0 +1,298 @@
+# Instancetypes and preferences
+
+**FEATURE STATE:** 
+
+* `v1alpha1` (Experimental) as of the [`v0.56.0`](https://github.com/kubevirt/kubevirt/releases/tag/v0.56.0) release
+
+## Introduction
+
+KubeVirt's [`VirtualMachine`](https://kubevirt.io/api-reference/master/definitions.html#_v1_virtualmachine) API contains many advanced options for tuning the performance of a VM that goes beyond what typical users need to be aware of. Users have previously been unable to simply define the storage/network they want assigned to their VM and then declare in broad terms what quality of resources and kind of performance characteristics they need for their VM.
```

Comment:
Can you point the links to the `main` branch, instead of the old `master` branch?